Source/WebCore/page/Navigator+LoginStatus.idl

```diff
@@ -28,7 +28,6 @@
     EnabledBySetting=LoginStatusAPIEnabled,
     ImplementedBy=NavigatorLoginStatus
 ] partial interface Navigator {
-    Promise<undefined> setLoggedIn();
-    Promise<undefined> setLoggedOut();
+    Promise<undefined> setLoginStatus(DOMString status);
     Promise<boolean> isLoggedIn();
```

Comment:
Similar here, it looks like this was changed since John added this. This should just be an attribute named `login`.

See https://fedidcg.github.io/FedCM/#login-status-javascript

Reply:
Actually, let's change this function to match the spec in a future PR, since its implementation may change after moving to persistent storage.